pkg/generator/templates/cloud-init-ubuntu.template

```diff
@@ -42,6 +41,7 @@ write_files:
 {{- end -}}
 {{- end }}
 runcmd:
+- until apt-get update -qq && apt-get install -qqy docker.io socat nfs-common logrotate jq policykit-1; do sleep 1; done
```

Comment:
Hm, interesting, didn't know that we are installing stuff via `apt-get`. How does this work in private/fenced environments?
I guess we can merge it now, but do we have a plan for the mentioned use-case?

Reply:
This extension is targeting the public infrastructures and the images available there. By default, docker is not installed on any of them, so we must do it with the extension. 
I've added the flag  `--no-upgrade`, so it will not try to upgrade already installed packages. 

Now there are three alternative options for private environments:
1. The landscape operator build a custom image with these packages preinstalled.  
1. An appropriate apt repositories is configured so that it can be used by the VMs from the environment to installed the required packages. 
1. Implement another os-ubuntu-* extension just like [alicloud](https://github.com/gardener/gardener-extension-os-ubuntu-alicloud/)

Reply:
👍 Thanks!